src/main/java/bio/overture/ego/service/AbstractPermissionService.java

```diff
@@ -13,14 +13,14 @@
 import static bio.overture.ego.utils.PermissionRequestAnalyzer.analyze;
 import static com.google.common.base.Preconditions.checkState;
 import static com.google.common.collect.Maps.uniqueIndex;
-import static com.gs.collections.impl.factory.Sets.intersect;
 import static java.util.Arrays.stream;
 import static java.util.Collections.reverse;
 import static java.util.Comparator.comparing;
 import static java.util.Objects.isNull;
 import static java.util.function.Function.identity;
 import static java.util.stream.Collectors.groupingBy;
 import static java.util.stream.Collectors.toMap;
+import static org.eclipse.collections.impl.factory.Sets.intersect;
```

Comment:
We don't need eclipse libraries to figure out intersections. We can use google guava which we already have on the classpath. 

```java
import com.google.common.collect.Sets;
```
And then later:

```java
Sets.intersection(...)
```